flyteidl/protos/flyteidl/core/workflow.proto

```diff
@@ -134,6 +134,11 @@ message ArrayNode {
         // to determine when an ArrayNode can be marked successful.
         float min_success_ratio = 4;
     }
+
+    oneof execution_version_option {
```

Comment:
Is there a reason to make this `oneof`? non-existence defaults to nil, but we could just as easily use the default of 0 for not being set I think?

Reply:
Complete brain fart on my end. Reverting the one_of change.

Reply:
I think the problem I noticed was if flyteidl isn't bumped on flytekit and then you pass in execution_version on the flytekit end. Answer would be to just ensure flyteidl version is bumped as part of that flyetkit change. Don't know why I did the one_of :face_palm: